### DISCUSSION TITLE
Mini.Colors: Add ready-to-go recipes in interactive()

### DISCUSSION THREAD

**Discussion by wroyca** [nvim-mini/mini.nvim#510] [Ideas]
Hello o/ 

Mini.colors is excellent for making quick adjustments to current colorscheme based on environment, (e.g., such as outdoors). That said, I often find myself spending too much time trying to remember specific configurations - I almost always have to open a new split and navigate to :h MiniColors, then search for the /recipes section. 

It would be neat to have these recipes right away in the interactive buffer, allowing users to make immediate adjustments by uncommenting lines. This would also allows users to save split for :hi instead of :h MiniColors.

Let me know what you think!

**Comment by echasnovski** [comment]
This sounds interesting.

Do you have particular recipes you find yourself using most often?

**Reply by wroyca** [reply]
Often, lightness and saturation, depending on my mood and how tired my eyes are, I like to reduce/increase them or simply switch from dark to light, among others. Sometimes temperature as well, and more rarely-but every now and then-hue if I feel the itch for a change.

**Reply by echasnovski** [reply]
Sound reasonable.

I created a #511 to track this. Thanks for suggestion!
